observium/firstrun.sh

```diff
@@ -17,6 +17,9 @@ else
   sed -i -e 's/USERNAME/observium/g' /config/config.php
 fi
 
+# if syslog config line is not already in config.php then add it
+grep -<span class="x x-first x-last">qF</span> 'enable_syslog' /config/config.php || echo "\$config['enable_syslog'] = 1;" &gt;&gt; /config/config.php
```

Comment:
The change was committed with markup.

Reply:
Oh, thats so frustrating, I used the "commit change" button in the PR!

Reply:
@charlescng should be ok now.